vignettes/introduction-to-partition.Rmd

```diff
@@ -69,56 +71,90 @@ df <- simulate_block_data(
+```
+
+Many types of data follow a pattern like this. Closely related to the block correlation structure found in genetic data is that found in microbiome data. The data set `baxter_otu` has microbiome data on 172 healthy patients. Each row represents a patient, and each column represents an Operational Taxonomic Unit (OTU). OTUs are species-like relationships between bacteria determined by analyzing their RNA. Each cell in the data represents the logged-count of the OTU found in the patient's stool sample, and there are 1,234 OTUs in all.
+
+```{r}
+baxter_otu
+```
+
+While not as apparent as simulated data, blocks of correlation also appear in these data; these bacteria tend to group together or apart in the microbiomes of participants. Here are the first 200 OTUs:
 
-df
+```{r}
+correlation_subset <- corr(baxter_otu[, 1:200])
+ggcorrplot::ggcorrplot(correlation_subset, hc.order = TRUE) + ggplot2::theme_void()
 ```
 
-partition takes a data frame and a threshold--the minimum amount of information a reduced variable must explain to be created--and reduces the data to as few variables as possible. The threshold prevents information loss: each reduced variable must still explain at least that much.
+# Reducing data with partition
+
+Because there are many more features (OTUs) in these data than rows (patients), it's useful to reduce data to meaningful subsets to use in statistical modeling. The primary function in partition is `partition()`. `partition()` takes a data frame and a threshold--the minimum amount of information a reduced variable must explain to be created--and reduces the data to as few variables as possible. The threshold prevents information loss: each reduced variable must still explain at least greater than or equal to the information threshold.
 
 ```{r}
-prt <- partition(df, threshold = .5)
+prt <- partition(baxter_otu, threshold = .5)
 
 prt
 ```
 
-partition creates a `tibble` with the newly reduced variables:
+For the microbiome data, `partition()` reduced 158 of the OTUs to 63 reduced variables. The other 1,076 OTUs were not reduced because doing so would have removed too much information from the data. `partition()` creates a `tibble` with the newly reduced variables, as well as any variables that were not reduced, which you can get with `partition_scores()`:
 
 ```{r}
 partition_scores(prt)
 ```
 
-Each variable in the original data set maps to one reduced variable; in this partition, there are four reduced variables consisting of two to three variables each, as well as five of the original variables that did not get reduced because too much information would have been lost.
+In comparison, PCA produces a data set of the same dimensions of the original data, with each component representing multiple variables, and with variables usually mapping to several components. While components are organized by their informativeness (the first component explains the most variance), no original features are retained. 
```

Comment:
Comment: in PCA, all original variables always map to all new variables, unless it is sparse PCA, thus, the statement, "PCA produces a data set of the same dimensions of the original data, with each component representing multiple variables, and with variables usually mapping to several components" should be revised.
Maybe replace with, "PCA produces a dataset in which all original variables map to all new components".